operators/src/source/ogr_source/mod.rs

```diff
@@ -610,6 +610,64 @@ impl FeaturesProvider<'_> {
         Ok(())
     }
 
+    ///  Creates a time filter string.
+    fn create_time_filter_string(
+        time_type: OgrSourceDatasetTimeType,
+        mut time_interval: TimeInterval,
+        driver: &str,
+    ) -> Option<String> {
+        match driver {
+            "PostgreSQL" => {
+                let postgres_range: TimeInterval = // https://www.postgresql.org/docs/current/datatype-datetime.html, capped by max supported TimeInstance value
```

Comment:
where do the constants come from? I don't find them in the linked doc.

Reply:
They are the low and high values from the timestamp with/without time zone in table 8.9

Reply:
converted to ms and then capped by the maximum suported TimeInstance value, because the GE won't be able to handle larger values anyways how I understand it